### PR TITLE
fix_maddpg_torch

### PR DESCRIPTION
**Pull request by yixin617**
add torch version

### CODE REVIEW

File: parl/core/torch/policy_distribution.py [hunk 1]
```diff
@@ -122,6 +121,74 @@ def kl(self, other):
             dim=1)
         return kl
 
+class DiagGaussianDistribution(PolicyDistribution):
+    """Categorical distribution for discrete action spaces."""
+
+    def __init__(self, logits):
+        """TODO
```
Comment (rical730) [inline parl/core/torch/policy_distribution.py:128]: please remove `TODO`

File: parl/algorithms/torch/maddpg.py [hunk 2]
```diff
@@ -103,7 +113,7 @@ def predict(self, obs, use_target_model=False):
         Args:
             obs (paddle tensor): observation, shape([B] + shape of obs_n[agent_index])
```
Comment (rical730) [inline parl/algorithms/torch/maddpg.py]: please replace `paddle tensor` with `torch tensor` in this file.

**Comment by rical730** [comment]
please modify benchmark/torch/maddpg referring to  https://github.com/PaddlePaddle/PARL/pull/828

**Comment by yixin617** [comment]
github bug

**Pull request closed without merging**